newrelic.yml

```diff
@@ -0,0 +1,45 @@
+#
+# This file configures the New Relic Agent.  New Relic monitors Ruby, Java,
+# .NET, PHP, Python, Node, and Go applications with deep visibility and low
+# overhead.  For more information, visit www.newrelic.com.
+#
+# Generated March 14, 2019
+#
+# This configuration file is custom generated for Home_3660
+#
+# For full documentation of agent configuration options, please refer to
+# https://docs.newrelic.com/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration
+
+common: &default_settings
+  # Required license key associated with your New Relic account.
+  license_key: e833ffa37df9d5af56540553d736145a9299c1bd
```

Comment:
do it through env var and add to docker-compose

Reply:
+